.github/workflows/cleanup-dp.yml

```
@@ -1,8 +1,9 @@
 name: Cleanup Deploy Previews
 
 on:
-  schedule:
-    - cron: 0 0 * * *
+  repository_dispatch:
```

This is for manually running this, right? Then you need `workflow_dispatch` here. 🙂

```suggestion
  workflow_dispatch:
```